csvProcessor/src/main/java/org/incredible/csvProcessor/CertModelFactory.java

```diff
@@ -53,6 +63,43 @@ private Class getParamType(String methodName) {
         } else
             return parameterTypes[0];
     }
+
+
+    private String calculateExpiryDateOfCertificate(String expiryDate, String issueDate) {
```

Comment:
Create a class called "ExpiryDateValuator" that has constructor taking up issueDate. Then move this logic to a function, 
`String getExpiryDate(String expiryDateValue)`

Also, as the sheet indicates, it must work for lowercase and uppercase 2m and 2M.